### DISCUSSION TITLE
How to change default font in PDF template maker

### DISCUSSION THREAD

**Discussion by OkayashiM** [YetiForceCompany/YetiForceCRM#16688] [How to]
I tried to change default font in PDF but for some reason cannot get expected outcome, so I would like to ask for guide as to replace Dejavu with my custom and set it as a default font in PDF templater

Thanks
おかし/Okayashi

**Comment by s2b-git** [comment]
I second it! Hoping for help too :-) Struggling with  #16682
Understand well that everything can be done by hacking core, but it would be good for the whole project if little more flexibility can be included in the box and shipped to community :)
